.classpath

```diff
@@ -1,28 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
ah yea for sure. I need to remember how to add that to the gitIgnore, thats what I do yea?